src/components/MovieCard/MovieCard.scss

```
@@ -1 +1,9 @@
-// 
+.card-image img {
```

```suggestion
.card__image img {
```

set `object-fit` property to card image, pay attention to your class name 

and then you can remove redundant styles for `.image`

---

I understand, thank you